guides/v2.1/cloud/cdn/fastly-vcl-wordpress.md

```diff
@@ -11,62 +11,127 @@ functional_areas:
+
+  -  `content`: When the VCL code in this example runs, it extracts the first part `mypath` of the path `/mypath/someotherpath`, and then compares that path to the values in the `wordpress_urls` dictionary. If a match is found, the request is redirected to the WordPress backend. See the [Fastly VCL reference](https://docs.fastly.com/vcl/reference/) for information about creating Fastly VCL code snippets.
+ 
+
+You can add the custom VCL snippet to your Fastly service configuration from the Admin UI (requires Fastly module 1.2.58 or later). If you do not have access to the Admin UI, you can save the JSON code example in a file and upload it using the Faslty API. See [Creating a VCL snippet using the Fastly API](https://docs.fastly.com/vcl/vcl-snippets/using-regular-vcl-snippets/#via-the-api). 
+
+
+#### To add the custom VCL snippet from the Admin UI
+
+1.  From the Magento Admin UI, navigate to **Stores** > **Configuration** > **Advanced** > **System** > **Fastly Configuration**.
+
+1.  Expand the **Custom VCL snippet** section.
+
+1.  Click **Create Custom Snippet**.
+
+1.  Complete the snippet form as shown in the following figure:
+
+    ![Create VCL Snippet]
+
+1.  After the page reloads, click **Upload VCL to Fastly** in the *Fastly Configuration* section.
+
+1.  After the upload completes, refresh the cache according to the notification at the top of the page.
+
+Fastly validates the updated version of the VCL code during the upload process. If the validation fails, edit your custom VCL snippet to fix the issue. Then,  upload the VCL again.
 
-Review the following values for the code to determine if you need to make changes:
 
-* `name`: Name for the VCL snippet. For this example, we used the name `wordpress`.
-* `priority`: Determines the order VCL snippets call. You want to set the priority to 5 to immediately run and check for URLs that should be redirected. This priority runs the snippet immediately and before any of the uploaded and default Magento VCL snippets (magentomodule) that have a priority of 50.
-* `type`: For this VCL, we use `recv`, which places it in the vcl_recv subroutine by below the boilerplate VCL and above any objects.
-* `content`: The code that runs. The code extracts the first part `mypath` of the path `/mypath/someotherpath`.  It then compares that path against the Edge Dictionary `wordpress_urls`. If a match is found, the visitor is redirected to the Wordpress backend.
+{: .bs-callout .bs-callout-info}
+Instead of manually uploading custom VCL snippets, you can add snippets to the `$MAGENTO_HOME/var/vcl_snippets_custom` directory in your environment. Snippets in this directory upload automatically when you click *upload VCL to Fastly* in the Admin UI. See [Automated custom VCL snippets deployment](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/CUSTOM-VCL-SNIPPETS.html#automated-custom-vcl-snippets-deployment) in the Fastly-Magento module documentation. 
```

Comment:
As in the badreferrer file, we don't necessarily need to change the text $MAGENTO_HOME, as it's pretty self explanatory. However it's worth noting that the equivalent environment variable on the actual Magento cloud environments is $MAGENTO_CLOUD_APP_DIR